Dining/models.py

```diff
@@ -132,6 +131,16 @@ def is_open(self):
         """
         return timezone.now() < self.sign_up_deadline
 
+    def has_room(self):
+        """
+        Determines whether this dining list can have more entries
+        :return: Whether this list can get more entries
+        """
+        if self.is_open():
```

Comment:
Niet belangrijk, just fyi voor als je dit toevallig niet wist: dit kan ook als oneliner:

`return self.is_open() and self.diners.count() < self.max_diners`

Reply:
Ik wist het, niet bij stil gestaan. Weet niet waarom, ik doe het vaak genoeg XD